python/lsst/pipe/base/argumentParser.py

```diff
@@ -280,6 +280,8 @@ class ArgumentParser(argparse.ArgumentParser):
       before I do this checking. Constructing a butler is slow, so I only want do it once,
       after parsing the command line, so as to catch syntax errors quickly.
     """
+    requireOutput = True  # Require an output directory to be specified?
+
```

Comment:
Could you please say a word or two about why this is a class variable instead of a constructor argument? I would have expected the latter.